### DISCUSSION TITLE
How change port 25 to 465 ssl

### DISCUSSION THREAD

**Discussion by brunoalvarenga** [postalserver/postal#2913] [Help with using Postal]
Hello

I install ssl, => https://docs.postalserver.io/features/smtp-tls  but 25 cant use ssl. im using 3.3.2.

How how change port 25 to 465 and add option to use too 587?

Thanks you

**Comment by willpower232** [comment]
Postal only supports explicit STARTTLS so connections have to start unsecure and upgrade.

If you try to use the other kinds that are secure from the beginning, it will not work as Postal is not currently compatible.

You can change which port Postal uses (or use iptables or multiple containers to listen on multiple ports) but if that implies STARTTLS then it won't work and whatever you're using will be confused.
